_data/toc.yaml

```diff
@@ -1392,11 +1392,14 @@ manuals:
       title: Hello Build
     - sectiontitle: Buildx
       section:
-        - path: /build/buildx/
-          title: Working with Buildx
         - path: /build/buildx/install/
           title: Install Buildx
-
+        - path: /build/buildx/
+          title: Working with Buildx
+        - path: /build/buildx/multiple-builders/
+          title: Using multiple builders
+        - path: /build/buildx/multiarch-images/
```

Comment:
We should be consistent:

```suggestion
        - path: /build/buildx/multi-platform-images/
```

Needs to rename `multiarch-images.md` to `multi-platform-images.md`

Reply:
It's true. It's called "multi-arch" in Desktop docs, and in conversation we do say multi-arch all the time.
I think most users would equate multi-platform with macOs/Linux/Windows. 
And architectures is clear that it refers to HW such as `arm`, `mips`, `ppc64le`, and even `s390x`.

Also,  taking for example this sentence in the original content:
"When you invoke a build, you can set the --platform flag to specify the target platform for the build output, (for example, linux/amd64, linux/arm64, or darwin/amd64)" or this command example "docker buildx build --platform linux/amd64,linux/arm64 "? 
We also say "An experimental package is available for arch-based distributions."  in Engine doc set.

Just trying to point out that it seems to be a combination of both. Platform is not enough to determine the arch (following the above interpretation): imho multi-architecture being more specific is a better candidate.

Reply:
Hum right, WDYT @tonistiigi @jedevc?

Reply:
I've noticed this inconsistency as well, agreed it would be good to clarify this, but maybe out of scope for this PR, there's a *lot* of build docs that mix-and-match between the two, so I'd propose to pick something here that works, and then discuss separately later.

Personally, I think multi-platform is better when talking about building images. A couple reasons:
- This is what the OCI spec refers to - the CPU architecture is part of the platform (the OS is the other main component). 
- Buildkit will eventually (hopefully) support building windows images. Say we have an `amd64` linux image and a `amd64` windows image - in this case, it's not as accurate to talk about multi-arch images, even though the underlying mechanism is the same. Multi-platform is a nice catch-all.
- The buildx/buildkit flags are `--platform` - calling it multi-arch seems like it can be confusing, multi-platform seems more inline with existing behavior.

Maybe there are still use-cases for talking about multi-arch though, for example, maybe around QEMU, when we're using linux throughout, but emulating different CPU architectures.

> An experimental package is available for arch-based distributions

I think this is probably unrelated, I think this refers to [ArchLinux](https://archlinux.org/), not multi-architecture/platform support.

Reply:
Agreed on the Engine example. I revisited it and @jedevc is right, but a simple docs search renders this
![image](https://user-images.githubusercontent.com/103652162/180989164-8cb03c38-80f3-4949-bb71-3a6dcaa312b2.png) 
![image](https://user-images.githubusercontent.com/103652162/180990266-904edc78-ea5f-48ff-b12f-857fee2e376a.png)
( some are community hits)

It seems for the discussion there's the case for both in content, when platform is the same and we do want to distinguish architectures, but that we'd like to introduce the feature as multi-platform as it's seen as capturing both.

But it leaves the case of harmonizing the remaining docs content.